packages/volto/src/components/theme/View/LinkView.jsx

```diff
@@ -52,6 +38,27 @@ const LinkView = ({ token, content }) => {
       )}
     </Container>
   );
+}
+
+const LinkView = ({ token, content }) => {
+  const history = useHistory();
+  useEffect(() => {
+    if (!token) {
+      const { remoteUrl } = content;
+      if (isInternalURL(remoteUrl)) {
+        history.replace(flattenToAppURL(remoteUrl));
+      } else if (!__SERVER__) {
+        window.location.href = flattenToAppURL(remoteUrl);
+      }
+    }
+  }, [content, history, token]);
+  if (__SERVER__ && !token && content.remoteUrl) {
+    return <Redirect to={content.remoteUrl} />;
+  }
+
+  const LinkViewBody = config.getComponent({ name: 'LinkViewBody' }).component;
+
+  return <LinkViewBody content={content} />;
```

Comment:
I would pass all the props down, in case there's something else which is useful.

Reply:
Done, thanks for the suggestion!